upstream-community-operators/postgresql-operator-phases-1-2-3/0.0.1/postgresql-operator-phases-1-2-3.v0.0.1.clusterserviceversion.yaml

```diff
@@ -0,0 +1,338 @@
+                image: quay.io/deekshah86/postgresql-operator
+                imagePullPolicy: Always
+                name: ansible
+                resources: {}
+                volumeMounts:
+                - mountPath: /tmp/ansible-operator/runner
+                  name: runner
+                  readOnly: true
+              - env:
+                - name: WATCH_NAMESPACE
+                  valueFrom:
+                    fieldRef:
+                      fieldPath: metadata.annotations['olm.targetNamespaces']
+                - name: POD_NAME
+                  valueFrom:
+                    fieldRef:
+                      fieldPath: metadata.name
+                - name: OPERATOR_NAME
+                  value: postgresql-operator
+                - name: ANSIBLE_GATHERING
+                  value: explicit
+                image: quay.io/deekshah86/postgresql-operator
+                imagePullPolicy: Always
+                name: operator
+                resources: {}
+                volumeMounts:
+                - mountPath: /tmp/ansible-operator/runner
+                  name: runner
+              serviceAccountName: postgresql-operator
+              volumes:
+              - emptyDir: {}
+                name: runner
+      permissions:
+      - rules:
+        - apiGroups:
+          - ""
+          resources:
+          - pods
+          - services
+          - services/finalizers
+          - endpoints
+          - persistentvolumeclaims
+          - events
+          - configmaps
+          - secrets
+          verbs:
+          - '*'
+        - apiGroups:
+          - apps
+          resources:
+          - deployments
+          - daemonsets
+          - replicasets
+          - statefulsets
+          verbs:
+          - '*'
+        - apiGroups:
+          - monitoring.coreos.com
+          resources:
+          - servicemonitors
+          verbs:
+          - get
+          - create
+        - apiGroups:
+          - apps
+          resourceNames:
+          - postgresql-operator
+          resources:
+          - deployments/finalizers
+          verbs:
+          - update
+        - apiGroups:
+          - ""
+          resources:
+          - pods
+          verbs:
+          - get
+        - apiGroups:
+          - apps
+          resources:
+          - replicasets
+          verbs:
+          - get
+        - apiGroups:
+          - postgresql.example.com
+          resources:
+          - '*'
+          verbs:
+          - '*'
+        - apiGroups:
+          - batch
+          resources:
+          - cronjobs
+          - jobs
+          verbs:
+          - '*'
+        serviceAccountName: postgresql-operator
+    strategy: deployment
+  installModes:
+  - supported: true
+    type: OwnNamespace
+  - supported: true
+    type: SingleNamespace
+  - supported: false
+    type: MultiNamespace
+  - supported: true
+    type: AllNamespaces
+  maturity: alpha
+  replaces: postgresql-operator-phases-1-2-3.v0.0.0
```

Comment:
This version of the Operator does not exist. Hence loading your Operator into a catalog fails because it checks whether you are supplying a valid update. If no previous version you can/want to update from exist, omit `replaces`.

Reply:
thanks will moify it

Reply:
Signed-off-by: Deeksha Hegde deeksha.h@gmail.com This is the sign used for my previous commits. Not sure why its failing for this one.

Reply:
removed replaces. Still stuck with this sign thing. Not sure why it wont work.

Reply:
Regarding signature, you have 23 commits and some are not signed. Please rebase (on master) and you will be asked which commit to pick. PlLease pick newest one and squash rest of them. Then you have just one commit which you can sign.

Reply:
Hello. I dont think i have good understanding of GIT repo. Could you share the commands i need to execute to ensure i dont mess up. I am blank right now looking at soo many commits i got. Kindly help with sequence and list of commands. :)